tensorflow/BUILD

```diff
@@ -643,6 +643,7 @@ tf_cc_shared_object(
         "//tensorflow/core/grappler/optimizers:custom_graph_optimizer_registry_impl",
         "//tensorflow/core:lib_internal_impl",
         "//tensorflow/core/profiler:profiler_impl",
+        "//tensorflow/core:training_op_helpers",
```

Comment:
@Squadrick Do we need this target in here. Removing this would get rid of "Multiple OpKernel registrations".